test/unit/org/apache/cassandra/db/commitlog/CommitLogSegmentManagerCDCTest.java

```diff
@@ -91,7 +90,7 @@ public void testCDCWriteFailure() throws Throwable
 
             // Simulate a CDC consumer reading files then deleting them
             for (File f : new File(DatabaseDescriptor.getCDCLogLocation()).tryList())
-                FileUtils.deleteWithConfirm(f);
+                f.deleteIfExists();
```

Comment:
There are multiple repetitions of this loop and of the creation of the array of CDC raw files across the test. We could encapsulate them in reusable methods:
```java
private static File[] getCDCRawFiles()
{
    return new File(DatabaseDescriptor.getCDCLogLocation()).tryList();
}

private static void deleteCDCRawFiles()
{
    for (File f : getCDCRawFiles())
    {
        f.deleteIfExists();
    }
}
```

Reply:
+1